aries-site/src/pages/foundation/wcag-checklist.mdx

```diff
@@ -0,0 +1,85 @@
+import { Checklist } from '../../components';
+
+There are three levels of compliance in the Web Content Accessibility Guidelines (WCAG) which reflect the priority of support:
+
+1. **A: Essential**
+   - If this is not met, assistive technologies will not be able to aid in reading, understanding or operating the page or product.
+2. **AA: Ideal Support**
+   - Required for multiple government and public websites. HPE strives for Level A and AA coverage on all products.
+3. **AAA: Specialized Support**
+   - Usually reserved for websites and applications that serve a specialized audience.
+
+### Success Criteria
+
+Each item on the checklist refers to a specific, testable success criterion within the WCAG 2.1 described by a reference number and title. Some accessibility issues align with multiple rules. In these cases, the most relevant criterion is identified for the checklist item.
+
+### Does this checklist guarantee a product is fully accessible?
+
+No. There is no such thing as a "100%" accessible product. However, this checklist prompts designers and developers to check for coverage of a wide range of disability conditions. Addressing these issues will improve experiences for all users who access your product and increase overall accessibility.
+
+## Global Code
+
+Global code affects an entire website or product application.
+
+export const globalCodeOptionsObjects = [
+  {
+    value: 'Validate your HTML by avoiding significant parsing errors.',
```

Comment:
What are we using for reference to create this list? Can you share where you are sourcing / getting inspiration from?

The presentation order seems a bit jumbled and isn't flowing for me. Can we group / order in a more intuitive manner? I am having trouble following.

Reply:
Here is the inspiration: https://www.a11yproject.com/checklist/

For this section in particular, I am taking from their grouping of items that relate to "global code" or the code that affects the entire page or product. These are usually easy and fast things to check first for overall functionality. Ensuring these things are in order would be a good first pass at the accessibility of the site.

Moving forward, the next sections will continue to draw inspiration from this checklist but I am cross referencing with more checklists that I have looked into to determine the best way to break up the WCAG: 
- https://webaim.org/standards/wcag/WCAG2Checklist.pdf
- https://tetralogical.com/articles/wcag-primer/
- https://boia.docsend.com/view/vwmnvs8?utm_medium=email&_hsmi=172511982&_hsenc=p2ANqtz-86ezk11520hPmuK4KogGfn51aCwbHoI-CF1iKl1uCgJA37sRXrIm4BLegl6y6uIbxcacWA29d2fUPnI5Ps8KOiYe2DFg&utm_content=172511982&utm_source=hs_automation
- https://www.w3.org/WAI/WCAG21/Understanding/

Reply:
I'd be interested in seeing the bullet-pointed "outline" of the checklist to get a sense of the vision. It doesn't (nor shouldn't) have all the detail at this stage, rather it serves as a roadmap for what this checklist might look like.